user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+administrators, and can be used when :ref:`creating or editing a user record <user-management-users>`. 
+
+The capability does not influence the user's ability to 
+:ref:`edit own password <user-guide-getting-started-profile-password>` from the *"My User"* page.
+
+
+.. _admin-capabilities-merge:
+
+Merge entities
+--------------
+
+The capability enables users to :ref:`merge <user-guide-ui-components-grids-delete-merge>` several records of the same 
+entity. This is mostly applicable to the merge of :ref:`accounts <user-guide-accounts>`.
+
+The capability is usually used by the sales representatives, marketing associates, and other employees working with 
+accounts.
+
+
+.. _admin-capabilities-outlook:
+
+Outlook integration
+-------------------
+
+The capability is available for enterprise users only. If this capability is set to *"System"*, users with the roles 
+will be able to :ref:`download <user-guide-getting-started-my-user-outlook>` the latest version 
+of OroCRM add-in for Outlook. 
+
+If :ref:`OroCRM is synchronized with Outlook <user-guide-synch-outlook>`, you can use the add-in to synchronize 
+contacts, tasks, and calendar events between OroCRM and your Outlook. You can associate emails to accounts, contacts, 
+opportunities, and cases and create leads, opportunities and cases in OroCRM based on emails from the Outlook side bar.
+
+This is a general capability that can improve the user experience for all enterprise users using both OroCRM and 
+Outlook. 
+
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+
+.. _admin-capabilities-address-dic:
+
+Read address dictionaries
+-------------------------
+
+The capability enables access to countries, regions, and address types via API. It has to be set to *"System"*, if 
+the Lead creation via Outlook should be supported. This capability should be available to system administrators or 
+integrators, authorized to access OroCRM via API.
+
+
+
+.. _admin-capabilities-search:
+
+Search
+------
+
+The capability enables users to use the :ref:`search <user-guide-getting-started-search>` 
+functionality to find specific records quickly.
+
+This is a general capability that can improve the overall user-experience.
+
+The setting does not influence the user's ability to :ref:`search by tag <user-guide-getting-started-search-tag>`.
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+The capability enables users to :ref:`send emails within OroCRM email campaigns <user-guide-email-campaigns-send>`.
+This way, marketing associates can sed a personalized template-based letter to multiple users.
+
+The setting does not effect the user's ability to create and edit :ref:`email campaigns <user-guide-email-campaigns>` 
+and should only be allowed to employees authorized to send email campaigns. These may be emails to potential customers, 
+other system users, etc.  
```

Comment:
Comment: It's unclear what what "These" refers to in the final sentence. Is it the type of emails in the email campaigns? Does this require clarification, or can this sentence be removed?